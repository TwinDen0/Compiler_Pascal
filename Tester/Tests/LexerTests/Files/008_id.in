aa_
_aa
a_a2