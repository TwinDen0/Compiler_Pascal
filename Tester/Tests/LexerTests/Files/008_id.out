1	1	IDENTIFIER	aa_	aa_
EndFile